lib/rules/images-no-direct-imports.js

```diff
@@ -0,0 +1,49 @@
+const {basename, dirname, extname} = require('path');
+const resolve = require('eslint-module-utils/resolve').default;
+const {docsUrl} = require('../utilities');
+
+function isImageImport(resolvedSource) {
+  return /\.(svg|png|jpg)$/.test(resolvedSource);
+}
+
+function isImportFromCurrentFolderIndex(contextFilename, resolvedSource) {
+  const isIndex =
+    basename(contextFilename, extname(contextFilename)) === 'index';
+
+  return isIndex && dirname(resolvedSource) === dirname(contextFilename);
+}
+
+module.exports = {
+  meta: {
+    docs: {
+      description:
+        "Prefer that imports of image files go through an index file within the image's directory instead of directly referencing images within a component.",
```

Comment:
A slight re-word maybe? `Prefer importing image files from a within a directory instead of from the path to the image file directly.`

Reply:
> Prefer importing image files from the index file of the directory instead of the direct path to the image file.